example/demo.py

```diff
@@ -479,6 +479,13 @@ def img_click_callback(event):
     Y = np.c_[i, j, k]
     viz.mesh(X=X, Y=Y, opts=dict(opacity=0.5))
 
+    # double y axis plot
+    viz = visdom.Visdom()
+    X = np.arange(20)
+    Y1=np.random.randinAt(0,20,20)
```

Comment:
`randinAt`? Probably intending to be `randint` as well right?

Reply:
oh... my typos.. I will make that change.